### ISSUE TITLE
/home/user/.local/share/Steam/steam.sh: line 444: no match: ssfn*

### ISSUE THREAD

**Issue by stephanieLGBT** [ValveSoftware/steam-for-linux#4680]
#### Your system information
- Steam client version: steam-1.0.0.53-1
- Distribution (e.g. Ubuntu): Arch Linux 64 bit
- Opted into Steam client beta?: No
- Have you checked for system updates?: Yes
#### Please describe your issue in as much detail as possible:

Expected steam to start.

Steam did not start.

Console output of 'steam': https://gist.github.com/TheSqrtMinus1/1c1dcb07eae454e1e5e9749f02dcfcb7

console output of 'strace steam': 
https://gist.github.com/TheSqrtMinus1/464bf7e0841d363f7710332dab92040f

list of packages installed: https://gist.github.com/TheSqrtMinus1/d8b317163f4d84aba57194fd0c341834
#### Steps for reproducing this issue:
1. Install 64 bit arch linux with stock xfce4 packages
2. Install Steam
3. Run Steam
4. Error

**Comment by her001** [comment]
After experiencing the same issue on Arch, I was seemingly able to work around it by switching to nouveau. Do you use the nvidia proprietary drivers? If so, you could try this.

I will do more testing on this in the future.

**Comment by txtsd** [comment]
I get this error on Arch with mesa.

**Comment by TheSqrtMinus1** [comment]
alright after uninstalling xf86-video-intel, steam-native works, but steam just gives me

```
libGL error: unable to load driver: i965_dri.so
libGL error: driver pointer missing
libGL error: failed to load driver: i965
libGL error: unable to load driver: i965_dri.so
libGL error: driver pointer missing
libGL error: failed to load driver: i965
libGL error: unable to load driver: swrast_dri.so
libGL error: failed to load driver: swrast
```

**Comment by her001** [comment]
@TheSqrtMinus1 That is completely normal, it happens for anyone using mesa on newer distros. It is caused by a library conflict between the Steam runtime and the system. Using steam-native is the proper way to work around this.

**Comment by secknv** [comment]
I have the exact same problem as first described in this issue on SparkyLinux 4.4 GameOver 64bit with the nvidia drivers. I am sort of a linux noob and there seems to be some mesa stuff installed on my system. Any advice?

**Comment by dobrowins** [comment]
As a temporary workaround I'd recommend to start from terminal ~/.local/share/Steam/ubuntu12_32/steam. It would update Steam (be patient! Might take few rounds of updates) and you'd be able to launch it as normal.
Worked in Ubuntu 17.10 Gnome.

**Comment by roachsinai** [comment]
@dobrowins Worked in Manjaro17.1 Xfce. Thanks.

**Comment by rhnvrm** [comment]
Incase anyone is using optirun, and faces this issue:

```
ILocalize::AddFile() failed to load file "public/steambootstrapper_english.txt".
[2018-02-17 22:26:56] Startup - updater built Nov 23 2016 01:05:42
Installing breakpad exception handler for appid(steam)/version(0)
SteamUpdateUI: An X Error occurred
X Error of failed request:  BadValue (integer parameter out of range for operation)
```

running @dobrowins solution with optirun seems to work.

`optirun ~/.local/share/Steam/ubuntu12_32/steam`

**Comment by 0xpr03** [comment]
@dobrowins solution still works here on KUbuntu 17.10

to clarify:
1 execute `~/.local/share/Steam/ubuntu12_32/steam` manually
2 run the normal launcher again
this will cycle through 2 update steps, afterwards I've had the normal login screen

**Comment by isaacbatst** [comment]
@dobrowins  Worked on Xubuntu 18.04

**Comment by LuisAlcaras** [comment]
Failed in Ubuntu MATE 18.04!

**Comment by kisak-valve** [comment]
Hello, Steam's handling of the Steam runtime changed a while back to prefer host libraries over the Steam runtime variants, and that was extended to the bootstrapper environment in the 1.0.0.55 Steam package.

Is anyone still experiencing this issue on an up to date system (with the 1.0.0.55 steam package or newer)?

**Comment by 0xpr03** [comment]
I haven't had it anymore since then

**Comment by kisak-valve** [comment]
Thanks for the feedback, closing as fixed with the 1.0.0.55 steam package.
